tools/place_name_resolver/README.md

```diff
@@ -12,12 +12,22 @@ includes local-id references to contained-in places.
 
 For sample input/output CSVs, see the `.csv` files in `place_name_resolver/testdata` directory.
 
+NOTE: If the `--generate_place_id` is set, then in place of DCIDs, Maps placeIDs
```

Comment:
i don't have too much context for this use, does it mean user could get the place id in batch (would that be a concern..)?

Reply:
This is no different than accessing the public Maps API which return place IDs, which is what the tools does, right?  The user of the tool anyway needs to provide the Maps API Key.

Reply:
Then that's okay.